### PR TITLE
fix: scheduler enabled tooltip

### PR DESCRIPTION
**Pull request by rephus**
Closes: #9041
### Description:

image
Screenshot from 2024-02-21 15-16-26

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/features/scheduler/components/SchedulersListItem.tsx [hunk 1]
```diff
@@ -61,12 +61,22 @@ const SchedulersListItem: FC<SchedulersListItemProps> = ({
...
-                        }}
-                    />
+                    <Tooltip
+                        label={
+                            scheduler.enabled
+                                ? 'Toggle off to temporarily pause notifications'
+                                : 'Notifications paused. Toggle on to resume'
+                        }
+                    >
+                        <div>
```
Comment (rephus) [inline packages/frontend/src/features/scheduler/components/SchedulersListItem.tsx:71]: required div for the toggle to work... is there a better way ?

**Comment by PriPatel** [comment]
Looks good to me👍

File: packages/frontend/src/features/scheduler/components/SchedulersListItem.tsx (hunk 1 above)
Comment (magnew) [reply packages/frontend/src/features/scheduler/components/SchedulersListItem.tsx:71]: You can/should use Box, but div is ok. Mantine wants a non-disabled ref.

**Review by magnew — APPROVED** [review]
This looks good!

File: packages/frontend/src/features/scheduler/components/SchedulersListItem.tsx (hunk 1 above)
Comment (rephus) [reply packages/frontend/src/features/scheduler/components/SchedulersListItem.tsx:71]: thanks for the tip. I updated the code

**Pull request merged**